test/Sentry.Serilog.Tests/Sentry.Serilog.Tests.csproj

```diff
@@ -4,8 +4,18 @@
     <TargetFrameworks>netcoreapp3.1;netcoreapp2.1;net461</TargetFrameworks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="2.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Primitives" Version="2.1.1" />
+    <PackageReference Include="Serilog" Version="2.7.1" />
```

Comment:
Right now the `Serilog` dependency comes in through the project reference:

`<ProjectReference Include="../../src/Sentry.Serilog/Sentry.Serilog.csproj" />`

Is this explicit one here needed?